backend/battling/views.py

```diff
@@ -53,17 +55,46 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class DeleteTeam(DeleteView):
-    template_name = "battling/delete_team.html"
+class DeleteBattle(DeleteView):
+    template_name = "battling/delete_battle.html"
     success_url = reverse_lazy("home")
-    queryset = Team.objects.all()
+    queryset = Battle.objects.all()
```

Comment:
The user can only delete only their battles, right?
If that's the case, then you should have a `get_queryset` and filter for battles where the user is a creator (or maybe if is an opponent too)